news/2024/2024-11-28-makeship-x-osu-plushies-second-run.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "Makeship x osu!: Limited Edition Plushies 2.0"
```

Comment:
thoughts on using a more varied title because the relevant info is already in the banner

```suggestion
title: "Another Batch of osu! Plushies"
```

Reply:
good idea